### DISCUSSION TITLE
Text-Shortcuts

### DISCUSSION THREAD

**Discussion by surfer-09** [guilryder/clavier-plus#38] [Q&A] [answered]
Hello,
I would like to use shortcuts like "#greeting" to trigger a text in Clavier+. Is it possible to set up such shortcuts? I haven't found a way to configure them yet.

Thanks!
Dennis

![Hotkey](https://github.com/guilryder/clavier-plus/assets/106930606/0b542038-a010-42d2-97b7-405965467fc4)

**Comment by guilryder** [comment, marked as answer]
No, Clavier+ can only create shortcuts for a single keystroke at a time. "#greeting" isn't a single keystroke.
